client/blocks/signup-form/index.jsx

```diff
@@ -957,6 +965,16 @@ class SignupForm extends Component {
 			);
 		}
 
+		if ( this.props.currentUser ) {
+			return (
+				<ContinueAsUser
+					redirectPath={ this.props.redirectToAfterLoginUrl }
+					onChangeAccount={ this.handleOnChangeAccount }
+					isSignUpFlow={ true }
```

Comment:
The `={ true }` is redundant.

```suggestion
					isSignUpFlow
```